bridges/ZoneTelechargementBridge.php

```diff
@@ -18,57 +18,109 @@ class ZoneTelechargementBridge extends BridgeAbstract {
+		if($filter == 'both' || $filter == 'streaming') {
+			// Get the post content, on the dedicated streaming website
+			$htmlstreaming = getSimpleHTMLDOM(self::UNPROTECED_URI_STREAMING . $this->getInput('url'))
+				or returnServerError('Could not request Zone Telechargement.');
+			// Get the HTML element containing all the links
+			$streaminglinkshtml = $htmlstreaming->find('p[style=background-color: #FECC00;]', 1)->parent()->next_sibling();
+			// Get all streaming Links
+			$liststreaming = $streaminglinkshtml->find('a');
+			foreach($liststreaming as $elementstreaming) {
+				// Retrieve the episode number from the link text
+				$epnumber = explode(' ', $elementstreaming->plaintext)[1];
 
+				// Format the link and add the link to the corresponding episode table
+				$episodes[$epnumber]['streaming'][] = '<a href="' . $elementstreaming->href . '">'
+					. $this->showTitle . ' Episode ' . $epnumber . '</a>';
+			}
 		}
 
 		// Finally construct the items array
 		foreach($episodes as $epnum => $episode) {
-			$item = array();
-			// Add every link available in the episode table separated by a <br/> tag
-			$item['content'] = implode('<br/>', $episode);
-			$item['title'] = $this->showTitle . ' Episode ' . $epnum;
-			// As RSS Bridge use the URI as GUID they need to be unique : adding a md5 hash of the title element
-			// should geneerate unique URI to prevent confusion for RSS readers
-			$item['uri'] = self::URI . $this->getInput('url') . '#' . hash('md5', $item['title']);
-			// Insert the episode at the beginning of the item list, to show the newest episode first
-			array_unshift($this->items, $item);
+			// Handle the Direct Download links
+			if(array_key_exists('ddl', $episode)) {
+				$item = array();
+				// Add every link available in the episode table separated by a <br/> tag
+				$item['content'] = implode('<br/>', $episode['ddl']);
+				$item['title'] = $this->showTitle . ' Episode ' . $epnum . ' - Téléchargement';
+				// As RSS Bridge use the URI as GUID they need to be unique : adding a md5 hash of the title element
+				// should geneerate unique URI to prevent confusion for RSS readers
+				$item['uri'] = self::URI . $this->getInput('url') . '#' . hash('md5', $item['title']);
+				// Insert the episode at the beginning of the item list, to show the newest episode first
+				array_unshift($this->items, $item);
+			}
+			// Handle the streaming link
+			if(array_key_exists('streaming', $episode)) {
+				$item = array();
+				// Add every link available in the episode table separated by a <br/> tag
+				$item['content'] = implode('<br/>', $episode['streaming']);
+				$item['title'] = $this->showTitle . ' Episode ' . $epnum . ' - Streaming';
+				// As RSS Bridge use the URI as GUID they need to be unique : adding a md5 hash of the title element
+				// should geneerate unique URI to prevent confusion for RSS readers
+				$item['uri'] = self::URI . $this->getInput('url') . '#' . hash('md5', $item['title']);
```

Comment:
You can set it to `$item['uid']` and leave `$item['url']` as original without hash.